docs/pages/guides/dom-components.mdx

```diff
@@ -192,7 +192,31 @@ export default function App() {
 
 ## Measuring DOM components
 
-You may want to measure the size of a DOM component and report it back to the native side (for example, native scrolling). This can be done using a native action:
+You may want to measure the size of a DOM component and report it back to the native side (for example, native scrolling). This can be done using a `autoSize` prop or a manual native action:
+
+### Automatically with `autoSize` prop
+
+You need to use the `dom={{ autoSize: true }}` prop to measure the size of the DOM component automatically and resize the native view coresponsingly:
+
+```tsx App.tsx (native)
+import MyComponent from './my-component';
```

Comment:
```suggestion
import DOMComponent from './my-component';
```

Let's update this line here. From the code snippet below ("web"),  in `my-component` file, the name of the component exported is `DOMComponent`.